config/metrics/monitor.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             key: service-ca.crt
             name: metrics-reader
             optional: false
-        serverName: etcd-shield.etcd-shield.svc
```

Comment:
This should be correct, since we use the same certificate for serving both metrics and webhooks.

Reply:
you're right, great catch! reverted with 38a963c0b657ff7d92903b939f056bb0a3b0a9c1